server/mcp_server_organization/README.md

```diff
@@ -0,0 +1,179 @@
+# organization MCP Server
+
+| 版本 |         v1.0.0          |
+|:--: |:-----------------------:|
+| 描述 | 火山引擎的企业组织服务是您使用火山引擎时的多账号管理服务。企业组织维护了多账号的账号间关系，为其他服务提供企业上云的底层支撑能力。       |
+| 分类 | 工具类 |
+| 标签 | 成员、企业组织、管控策略、管理与治理           |
+
+## Tools
+本 MCP Server 产品提供以下 Tools:
+
+- CreateOrganization: 创建企业组织
+
+- CreateOrganizationalUnit: 创建企业组织单元
+
+- CreateServiceControlPolicy: 创建管控策略
+
+- DescribeAccount: 查询成员账号信息
+
+- DescribeAccountInvitation: 成员账号查询加入企业组织邀请信息
+
+- DescribeOrganization: 查询企业组织信息
+
+- DescribeOrganizationalUnit: 查询组织单元信息
+
+- DescribeQuitApplication: 查询成员账号退出申请信息
+
+- EnableServiceControlPolicy: 启用管控策略功能
```

Comment:
disable this tool by default and allow user to enable?
EnableServiceControlPolicy